includes/admin/class-wc-rest-payments-deposits-controller.php

```diff
@@ -60,8 +60,8 @@ public function register_routes() {
 	 * @param WP_REST_Request $request Full data about the request.
 	 */
 	public function get_deposits( $request ) {
-		$page      = intval( $request->get_params()['page'] );
-		$page_size = intval( $request->get_params()['pagesize'] );
+		$page      = (int) $request->get_params()['page'];
+		$page_size = (int) $request->get_params()['pagesize'];
```

Comment:
I'm not a PHP expert, but does that type conversion is supported in earlier versions of PHP? The minimum supported by the plugin version is 5.3.

Reply:
Yes: https://3v4l.org/Wsks4C (loooks like it was introduced even earlier then I though: PHP 4.3+)

Reply:
What about adding the requirement to composer.json (as 7.* while we are evaluating v8 compatibility)?

Reply:
Ok, will add it then. A bonus is also that PhpStorm will be able to recognize the language level properly.